### PR TITLE
Adding contributing file

### LINKED ISSUE #88: Document CONTRIBUTING
include documenting maintainers and the review flow.

### PR DESCRIPTION
**Pull request by mattfarina**
Closes #88

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+Then you add a line to every git commit message:
+
+    Signed-off-by: Joe Smith <joe.smith@example.com>
+
+Use your real name (sorry, no pseudonyms or anonymous contributions.)
+
+If you set your `user.name` and `user.email` git configs, you can sign your
+commit automaticallynwith `git commit -s`.
```
Comment (ericpromislow) [inline CONTRIBUTING.md:67]: typo between `automatically` and `with`

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Ready for merging after fixing the typo

File: CONTRIBUTING.md (hunk 1 above)
Comment (mattfarina) [reply CONTRIBUTING.md]: Thanks for catching that.

**Comment by mattfarina** [comment]
This is ready to be re-reviewed

**Review by ericpromislow — APPROVED** [review]
LGTM

**Pull request merged**